conf/airframes/tudelft/rot_wing_25kg.xml

```diff
@@ -393,6 +422,11 @@
     </section>
 
     <section name="GUIDANCE_INDI_HYBRID" prefix="GUIDANCE_INDI_">
+        <!--WLS settings-->
+        <define name="USE_WLS"              value="FALSE|TRUE"/>
```

Comment:
you let the compiler choose true or false ? ;)

Reply:
Funny indeed, but it worked.